### PR TITLE
Adding netbull's recipe

### PR DESCRIPTION
**Pull request by netbull**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: netbull/security-bundle/1.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "NetBull\\SecurityBundle\\NetBullSecurityBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
+        "vendor/bin/browscap-php browscap:fetch": "php-script",
```
Comment (Nyholm) [inline netbull/security-bundle/1.0/manifest.json]: Could you tell me why these lines are needed?

Comment (netbull) [reply netbull/security-bundle/1.0/manifest.json]: the idea is to use browscap for creating browser fingerprint and store which will be used to ban after X attempts and for X seconds. Browscap need to create his cache after installing and that's why the commands are needed.

Comment (Nyholm) [reply netbull/security-bundle/1.0/manifest.json]: Yeah, I understand that. But I do not see that your bundle uses browscap other then in composer.json.

Comment (netbull) [reply netbull/security-bundle/1.0/manifest.json]: Ahh, I'm still developing... after 30 min it will be ready.. :/

Comment (netbull) [reply netbull/security-bundle/1.0/manifest.json]: now it should be fine :)

**Comment by Nyholm** [comment]
Great. Can you make sure travis is happy? 
It seams like you depend on doctrine bundle.

**Comment by netbull** [comment]
The Travis should be happy /at least for Symfony 4/, but I can not make rebuild without useless commit...

**Pull request merged**